frontend/packages/ceph-storage-plugin/integration-tests/views/installFlow.view.ts

```diff
@@ -71,11 +72,13 @@ export const goToWorkLoads = async () => {
 export const selectWorkerRows = async () => {
   // Wait for 6 inputs to show up
   const selectedNodes = [];
-  await browser.wait(until.and(waitForCount($$('tbody tr'), 6)));
+
+  // await browser.wait(until.and(waitForCount($$('tbody tr'), 6)));
+  await browser.wait(until.presenceOf($('[data-label="Role"]')), 10000);
+  expect(await browser.wait($$('tbody tr').count())).toBeGreaterThanOrEqual(3);
```

Comment:
And number 3, do we have this defined somewhere in constants like we know that only 3 nodes are used, or is this value somehow configurable?  If this number is used on multiple places and I guess it is as we probably hardcode now only for 3 worker nodes we should use this value from constant I think.

The better approach is to have this somehow configurable in the future (different PR) as this will be needed to support more nodes I think.

Reply:
This is only a pre-check that we have nodes - it doesn't do nothing. The worker selection is done here:
https://github.com/openshift/console/blob/c505bd10c97343a84b5cb1519e87d848d4db2ad6/frontend/packages/ceph-storage-plugin/integration-tests/views/installFlow.view.ts#L81-L109

Reply:
The number 3 is used two times and once for all the nodes and the second is for only workers. Before my change it was 6 and 3 so I don't think we need to make it a const.